.github/PULL_REQUEST_TEMPLATE.md

```
@@ -20,13 +22,24 @@
 - [ ] Port
 - [ ] Translate
 - [ ] Resprite
-<!- Вы должны понимать, что несоблюдение вышеизложенного может привести к закрытию вашего PR по усмотрению разработчика ->
 
-**Список изменений**
+**Изменения**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)